### ISSUE TITLE
Make accessibility improvements

### ISSUE THREAD

**Issue by versatilehuman** [smartchicago/trailsy#124]
https://pages.18f.gov/accessibility/checklist/

**Comment by versatilehuman** [comment]
So most of this is either not applicable, not possible or we're doing it.

- We've addressed the trail color issue with the new pop-up info.
- We fail the contrast rules in places, so I'm going to test everything and adjust accordingly. 
- We pass the flashing rules with our little bit of flashing

We won't be able to make this accessible without an alternative version:

> Alternate versions of web pages should only be used when the main page can't be made accessible. There are very few instances where they would be necessary. **Live dynamic mapping applications are one example.**

The alternative version will be the updated website with this same information.

**Comment by versatilehuman** [comment]
Once the new fpdcc.com launches:

"Providing a link at the beginning of a nonconforming Web page that points to a conforming alternate version"
 https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G136
